docs/java/features/resilience/caching.mdx

```diff
@@ -0,0 +1,189 @@
+By default a cache entry is cleared once the configured duration has passed since the entry was created.
+
+Finally, parameters can be listed that further refine the caching functionality.
+Parameters values will be attached to cache key.
+When the cache is accessed only entries with matching parameter values will be returned.
+
+```java
+// Set a parameter value
+cacheConfig = CacheConfiguration
+        .of(Duration.ofDays(1))
+        .withParameters("value1");
+resilienceConfig.cacheConfiguration(cacheConfig);
+
+// Cache is empty, so this will be a cache miss
+ResilienceDecorator.executeSupplier(() -> operation(), resilienceConfig);
+// Cache is hit, cached value will be returned
+ResilienceDecorator.executeSupplier(() -> operation(), resilienceConfig);
+
+// Set a different parameter value
+cacheConfig = CacheConfiguration
+        .of(Duration.ofDays(1))
+        .withParameters("value2");
+resilienceConfig.cacheConfiguration(cacheConfig);
+
+// Cache is not hit because the existing cache entry has a different parameter value
+ResilienceDecorator.executeSupplier(() -> operation(), resilienceConfig);
+
+cacheConfig = CacheConfiguration
+        .of(Duration.ofDays(1))
+        .withParameters("value1");
+resilienceConfig.cacheConfiguration(cacheConfig);
+
+// The cached value from the very first invocation is returned as the parameter is "value1" again
+ResilienceDecorator.executeSupplier(() -> operation(), resilienceConfig);
+```
+
+So you can use parameters to separate cache entries as you see fit.
+Please note that there is no need to do this to achieve multi tenancy.
+The SDK is tenant aware by default as described below.
+
+### Multi Tenancy
+
+By default this caching functionality is tenant aware.
+If a tenant is available the SDK will create cache entries only for that specific tenant.
+Other tenants will not see the value cached for that specific tenant.
+
+For this example assume that `config` is a `ResilienceConfiguration` that has a `CacheConfiguration` in place:
+
+```java
+cachedOperation = ResilienceDecorator.decorateSupplier(() -> operation(), config)
+
+// cachedOperation() will be evaluated
+TenantAccessor.executeWithTenant(tenantA, () -> cachedOperation());
+// cached value will be taken
+TenantAccessor.executeWithTenant(tenantA, () -> cachedOperation());
+
+// cachedOperation() will be evaluated because the cached value is only accessible for tenantA
+TenantAccessor.executeWithTenant(tenantB, () -> cachedOperation());
+// cached value for tenantB will be taken
+TenantAccessor.executeWithTenant(tenantB, () -> cachedOperation());
+```
+
+This isolation is configured via the isolation mode of the resilience configuration.
+So you can disable tenant aware caching by setting the isolation mode to `NO_ISOLATION`.
+See the section on isolation levels under [Resilience](https://sap.github.io/cloud-sdk/docs/java/features/resilience/resilience#multi-tenancy)
+
+## Manipulating the State of Caches
+
+:::note
```

Comment:
I would move this note down and rather start by saying "you may manually clear caches".

The note is quite specific and only nice-to-know. In general documentation should go from general -> specific. Because general information is relevant for all readers whereas specific things may only be relevant for some. So you usually want the general information first.

Reply:
I had moved the note upwards.